core/tools/scripts/red-pill

```diff
@@ -1,5 +1,38 @@
+#!/bin/bash
+
+usage() {
+  echo "Usage: $0 [-h] [-u user]"
+  echo
+  echo "  -h,        Show this help message"
+  echo "  -u,        Define a user (default: pi)"
+  exit 1
+}
+
+# Default values
+user="pi"
+
+while getopts ":hu:" opt; do
+  case ${opt} in
+    h )
+      usage
+      ;;
+    u )
+      user=$OPTARG
+      ;;
+    \? )
+      echo "Invalid Option: -$OPTARG" 1>&2
+      usage
+      ;;
+    : )
+      echo "Invalid Option: -$OPTARG requires an argument" 1>&2
+      usage
+      ;;
+  esac
+done
+shift $((OPTIND -1))
+
```

Comment:
```suggestion

# Don't simply ignore additional arguments
if [ $# -gt 0 ]; then
  echo "Invalid Argument: $1" 1>&2
  usage
fi

```

Reply:
I'm not sure if I follow, what is the use case for this ?

Reply:
`redpill myusername` would print the usage, instead of ignoring the argument. It's pretty common for people to miss the argument identifier (say, `-u`) before the argument.

Reply:
Oh, nice!